xbmc/platform/darwin/ios-common/storage/IOSStorageProvider.mm

```diff
@@ -37,5 +37,20 @@
 
 std::vector<std::string> CIOSStorageProvider::GetDiskUsage()
 {
-  return {};
+  std::vector<std::string> result;
+  auto fileSystemAttributes = [NSFileManager.defaultManager attributesOfFileSystemForPath:@"/"
+                                                                                    error:nil];
+  
+  auto formatter = [NSByteCountFormatter new];
+  formatter.includesActualByteCount = YES;
+  formatter.zeroPadsFractionDigits = YES;
+  
+  for (auto pair :
+       {std::make_pair(NSFileSystemFreeSize, 160), std::make_pair(NSFileSystemSize, 20161)})
```

Comment:
for (const auto& pair : {{NSFileSystemFreeSize, 160}, {NSFileSystemSize, 20161}})

Reply:
* const & - added
* clang-format adds line break, don't think I should touch that